src/views/studio/MyUploadsView/MyUploadsView.style.tsx

```diff
@@ -1,11 +1,13 @@
 import styled from '@emotion/styled'
 
 import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
+import { TOP_NAVBAR_HEIGHT } from '@/components/TopbarBase'
 import { Text } from '@/shared/components/Text'
 import { sizes } from '@/shared/theme'
 
 export const UploadsContainer = styled(LimitedWidthContainer)`
   padding-bottom: 120px;
+  min-height: calc(100vh - ${TOP_NAVBAR_HEIGHT}px - ${sizes(8)});
```

Comment:
Why do we make this change?

Reply:
Hmm, I think that I wanted to check something and committed this by mistake. Will erase these lines.